apps/passport-client/.gitignore

```diff
@@ -5,3 +5,4 @@ public/js
 public/service-worker.js
 public/index.html
 public/bundle-stats.html
+public/engine
```

Comment:
perhaps worth naming this directory to include the word `strich` could be useful